modules/manage/pages/manage-settings/configure-compact-settings.adoc

```diff
@@ -181,9 +181,15 @@ The specified number of days will elapse before tombstones for expired or delete
 The default value is three days.
 The permitted range of values is `0.04` to `60` (where `0.04` equals one hour, and `1` equals one day.
 
-Note that if you set this value too low, you may see inconsistent results in Views queries, such as deleted items appearing in a result set.
+[CAUTION]
+====
+If you set this value too low, you may see inconsistent results in Views queries, such as deleted items appearing in a result set.
 You may also see inconsistent items across clusters, if XDCR has been set up between the clusters.
+
+If the metadata purge interval is set too low, it can also cause severe issues with transactions, especially for ephemeral buckets
```

Comment:
Full stop at end
```suggestion
If the metadata purge interval is set too low, it can also cause severe issues with transactions, especially for ephemeral buckets.
```